test/generators/webmock/install_generator_test.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require "test_helper"
+require "generators/boring/webmock/install/install_generator"
+
+class WebmockInstallGeneratorTest < Rails::Generators::TestCase
+  tests Boring::Webmock::InstallGenerator
+  setup :build_app
+  teardown :teardown_app
+
+  include GeneratorHelper
+  include ActiveSupport::Testing::Isolation
+
+  def destination_root
+    app_path
+  end
+
+  def test_should_install_webmock_successfully
+    Dir.chdir(app_path) do
+      install_rspec
+      quietly { run_generator [destination_root, "--test_gem=rspec"] }
+
+      assert_file "Gemfile" do |content|
+        assert_match(/webmock/, content)
+      end
+    end
+  end
+
+  def test_should_raise_error_for_invalid_option
```

Comment:
Let's split this test into two tests:

test_should_raise_error_for_blank_test_gem
test_should_raise_error_for_invalid_test_gem